src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Get a container registry and show result in a table
+            az acr show -n <registry-name> -o table
+
+Update a container registry
+-------------
+::
+
+    Command
+        az acr update: Update a container registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+        --tags              : Multiple semicolon separated tags in 'key[=value]' format.  Use "" to
+                            clear existing tags.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Update tags of a container registry and show result in a table
+            az acr update -n <registry-name> --tags key1=value1;key2=value2 -o table
+
+The catalog of repositories in the specified registry
+-------------
+::
+
+    Command
+        az acr catalog: The catalog of repositories in the specified registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+        --password          : The password used to log into the container registry.
+        --username          : The username used to log into the container registry.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        The catalog of repositories in a registry under the current subscription
+            az acr catalog -n <registry-name>
+        The catalog of repositories in any registry with credentials
+            az acr catalog -n <registry-name> --username <username> --password <password>
+
+The list of tags for a given repository in the specified registry
+-------------
+::
+
+    Command
+        az acr tags: The list of tags for a given repository in the specified registry.
```

Comment:
>Why are there no other commands relating to repositories?

The `catalog` command is also related to repositories. Catalog will tell you that you have things like 'ubuntu', 'hello-world' and 'myapp' in your registry.

> Just 'tags' implies to me that I could show tags on a container.

We're dealing with container repositories, so 'tags' does list the tags of a container repository. For example, you can list all the tags for the 'ubuntu' repository, and you'll get a result like '14.04' and '16.04'.

I do agree that we can do verb-noun command names if that's what other command names follow.